lib/Agrammon/UI/CommandLine.pm6

```diff
@@ -206,20 +231,26 @@ sub run (IO::Path $path, IO::Path $input-path, $technical-file, $variants, $form
     }
 }
 
+sub connect-db(Str $cfg-filename) {
+    my $cfg = Agrammon::Config.new;
+    $cfg.load($cfg-filename);
+    my $db = DB::Pg.new(conninfo => $cfg.db-conninfo);
+    PROCESS::<$AGRAMMON-DB-CONNECTION> = $db;
+    return ($cfg, $db);
```

Comment:
Given this both loads config and connects the DB, I wonder if a better name is in order...

Reply:
Renamed.